_docs/examples/provider/kustomization.yaml

```diff
@@ -1,15 +1,16 @@
 bases:
-  - github.com/ovrclk/akash/_docs/kustomize/akash-provider?ref=master
-  # - ../../kustomize/akash-provider
+  - ../../kustomize/akash-provider
```

Comment:
this requires you to be doing this in-repo, right?

Reply:
Yeah, I'll change it to default to the github URL, but leaving the local path is usefully for quickly getting back to a debugging environment.